README.md

```diff
@@ -483,6 +484,22 @@ The data format for donut, pie, and radial bar are the simplest. They are
 just any single value of type Numeric.
 
 
+## Options (Reference)
+ApexCharts.rb supports [all options from apexcharts.js](https://apexcharts.com/docs/options/). They should be written in snake_case instead of camelCase.
```

Comment:
Change:. 
`. They should be written in snake_case instead of camelCase.`
Into:. 
`, but instead of camelCase you can write it in snake_case.`

Reply:
"ApexCharts.rb also provides shortcuts to some ApexCharts.js options, such as `title`. In ApexCharts.js you would have to write `title:{ text: "Some title" }`. In ApexCharts.rb you can write `title: "Some title"` if you just want to add the text.  
`xtitle` and `ytitle` are even greater shortcuts. Instead of `xaxis: { title: { text: "x title" } }` you can write `xtitle: "x title"`."